src/use_hotkeys.rs

```diff
@@ -93,3 +99,43 @@ pub fn use_hotkeys(key_combination: &'static str, on_triggered: Callback<()>) {
     });
 
 }
+
+
+pub fn use_hotkeys_ref<T>(
+    key_combination: &'static str,
+    on_triggered: Callback<()>,
+) -> NodeRef<T>
+where T: ElementDescriptor + 'static + Clone
```

Comment:
interesting we have to implement the `Clone` trait

Reply:
Yeah, I don't know why they don't have it derived by default for all ElementDescs